reco_utils/dataset/movielens.py

```diff
@@ -18,18 +18,46 @@
         FloatType,
         DoubleType,
         LongType,
+        StringType,
     )
-except ModuleNotFoundError:
```

Comment:
Can you add back ModuleNotFoundError? this should pass when we are in the bare env but if there is another exception, we should know